pkgs/by-name/sl/slackdump/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildGo123Module,
+  lib,
+  fetchFromGitHub,
+}:
+
+buildGo123Module rec {
+  pname = "slackdump";
+  version = "2.5.11";
+
+  src = fetchFromGitHub {
+    owner = "rusq";
+    repo = "slackdump";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-othSIR+KyekUt+/LWhaM5Y6LGsujDCZTfLJotTcPXc0=";
+  };
+
+  vendorHash = "sha256-6GVXzaJ5Lrt4fF0c9cbx6AqpgCwlYgKf6jUJkUyxc0s=";
+
+  meta = {
+    homepage = "https://github.com/rusq/slackdump";
+    changelog = "https://github.com/rusq/slackdump/releases/tag/v${version}";
+    description = "Archive your private and public Slack messages.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

must not end in period